### PR TITLE
Add joi!

### PR DESCRIPTION
**Pull request by minsulee2**
Joi has just been adopted! Now, safely get request body!!

### CODE REVIEW

**Review by platfowner — APPROVED** [review]
LGTM
Thanks!

**Review by jiyoung-an — APPROVED** [review]
LGTM, thank you for implementing such a great schema tools!

File: index.js [hunk 1]
```diff
@@ -15,7 +15,7 @@ app.use(express.urlencoded({ extended: true }));
 app.use(express.json());
 
 // TODO(minsu): Add Joi
-app.post('/service', Middleware.classifyJobType, OpenaiAinizeHandler.service);
+app.post('/service', Middleware.classifyJobType, Middleware.joiValidate, OpenaiAinizeHandler.service);
```
Comment (jiyoung-an) [inline index.js:18]: Is the `Ainize.Middeleware.blockchainTriggerFilter` middleware function applied separately?

File: handlers/joi.js [hunk 2]
```diff
@@ -0,0 +1,126 @@
...
+    order: Joi.string().valid('asc', 'desc'),
+    after: Joi.string(),
+    before: Joi.string()
+  }),
+  retrieveAssistantSchema: Joi.object({
+    assistant_id: Joi.string().required()
+  }),
+  modifyAssistantSchema: Joi.object({
+    assistant_id: Joi.string().required(),
+    model: Joi.string().required(),
```
Comment (jiyoung-an) [inline handlers/joi.js:33]: model is optional parameter in openAI ([link](https://platform.openai.com/docs/api-reference/assistants/modifyAssistant#assistants-modifyassistant-model))

File: index.js (hunk 1 above)
Comment (minsulee2) [reply index.js:18]: This will be included next PR!

File: handlers/joi.js (hunk 2 above)
Comment (minsulee2) [reply handlers/joi.js:33]: Oh, thanks for finding this!

**Comment by minsulee2** [comment]
Thanks!!! Now Merging!

**Pull request merged**